### PR TITLE
working build

### PR DESCRIPTION
**Pull request by tg339**

### CODE REVIEW

File: apps/company-landing/app/blog/(posts)/moose-launch/@breadcrumbs/page.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Text } from "@514labs/design-system/typography";
+import { Text } from "@514labs/design-system-components/typography";
```
Comment (cjus) [inline apps/company-landing/app/blog/(posts)/moose-launch/@breadcrumbs/page.tsx:1]: Interesting there's now a design-system-components which is separate from the other design-system.

File: apps/company-landing/package.json [hunk 2]
```diff
@@ -19,7 +19,8 @@
     "@repo/ts-config": "workspace:*",
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.0",
-    "@514labs/design-system": "workspace:*",
+    "@514labs/design-system-base": "workspace:*",
```
Comment (cjus) [inline apps/company-landing/package.json:22]: I see.

**Review by cjus — APPROVED** [review]
98 files changed.  Are you F-N kidding me? :-D. I do see that this is a major rearchitecture and refactoring effort.  Really cool and I'm looking forward to perhaps having you share the experience during our next dev meeting!

File: apps/company-landing/app/blog/(posts)/moose-launch/@breadcrumbs/page.tsx (hunk 1 above)
Comment (tg339) [reply apps/company-landing/app/blog/(posts)/moose-launch/@breadcrumbs/page.tsx:1]: Yep, splitting the two libraries and making base re-usable across mono repos and components reusable in a mono-repo

**Pull request merged**